usaspending_api/api_contracts/contracts/v2/agency/toptier_code/object_class.md

```diff
@@ -65,23 +66,44 @@ Returns a list of Object Classes in the Agency's appropriations for a single fis
                     "hasPrevious": false,
                     "count": 10
                 },
+                "totals": {
+                    "obligated_amount": 350.0, 
+                    "gross_outlay_amount": 200.36
+                },
                 "results": [
                     {
-                        "name": "Grants, subsidies, and contributions",
-                        "obligated_amount": 12329194065.31,
-                        "gross_outlay_amount": 13196218848.88
-                    },
-                    {
-                        "name": "Full-time permanent",
-                        "obligated_amount": 221101112.31,
-                        "gross_outlay_amount": 230953611.8
+                        "name": "Personnel and compensation benefits",
+                        "obligated_amount": 350.0,
+                        "gross_outlay_amount": 200.36
+                        "children": [
+                            {
+                                "name": "Full-time permanent",
+                                "obligated_amount": 150.0,
+                                "gross_outlay_amount": 100.26
+                            },
+                            {
+                                "name": "Other than full-time permanent",
+                                "obligated_amount": 200.0,
+                                "gross_outlay_amount": 100.10
+                            }
+                        ]
                     }
                 ],
                 "messages": []
             }
 
 # Data Structures
 
+## Totals (object)
++ `obligated_amount` (required, number)
++ `gross_outlay_amount` (required, number)
```

Comment:
Should this include `total_budgetary_resources` or is that field unnecessary for this tab?